packages/toolbars-modals/web/src/link/LinkComponents/AdvancedCheckboxes.tsx

```diff
@@ -56,14 +56,14 @@ const AdvancedCheckboxes: React.FC<Props> = ({
               />
               {isMobile && useAdvancedSettings ? (
                 <div className={classNames(styles.linkPanel_advancedSettings_mobile_text_info)}>
-                  {t(checkbox.tooltipTextKey)} {renderLearnMoreLink()}
+                  {t(checkbox.tooltipTextKey)} {renderMoreLink()}
```

Comment:
```suggestion
                  {t(checkbox.tooltipTextKey)}
                  <MoreLink link={learMoreUrl} t={t} />
```

this split is not necessary, I this it is more readable that way